client/src/App.js

```diff
@@ -2,50 +2,49 @@ import React from 'react';
 import { useAuth0 } from "@auth0/auth0-react";
 import './App.css';
 import { Route, BrowserRouter as Router, Switch } from 'react-router-dom'
-import Profile from './Pages/Profile';
-import Experts from './Pages/Experts';
-import Learners from './Pages/Learners';
+import Profile from './pages/Profile';
+import Experts from './pages/Experts';
+import Learners from './pages/Learners';
+import Landing from './pages/Landing';
 
-function App() {
+const LoginButton = (props) => {
+  return (
+    <button onClick={props.onclick}>{props.children}</button>
+  )
+}
+
+
+const App = (props) => {
   const {
-    isLoading,
     isAuthenticated,
     error,
     user,
     loginWithRedirect,
     logout,
   } = useAuth0();
-
-  if (isLoading) {
-    return <div>Loading...</div>;
-  }
-  if (error) {
-    return <div>Oops... {JSON.stringify(error)}</div>;
-  }
-
-  if (isAuthenticated) {
+  const handleLogout = () => logout({ returnTo: window.location.origin })
```

Comment:
Add @TODO here which was in previous changes.

Reply:
Lmao whoops